packages/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -202,7 +202,7 @@ void main() {
         var secondRes = await Amplify.API.query(request: secondReq!).response;
         var secondData = secondRes.data;
         expect(secondData?.items.length, limit);
-        expect(secondData?.items[0].id, isNot(firstData?.items[0].id));
+        expect(secondData?.items[0]?.id, isNot(firstData?.items[0]?.id));
```

Comment:
```
expect(secondData?.items[0]?.id, isNot(firstData?.items[0]?.id));
```
can theoretically produce
```
expect(null, isNot(null));
```
? Should we add some sort of constraint to this `expect`?

Reply:
Good callout. I will add extra check to ensure both IDs are non-empty, non-null strings, in addition to ensuring they are not the same.